src/renderers/shared/stack/event/ReactSyntheticEventType.js

```diff
@@ -8,14 +8,29 @@
  *
  * Flow type for SyntheticEvent class that includes private properties
  *
- * @providesModule ReactSyntheticEvent
+ * @providesModule ReactSyntheticEventType
  * @flow
  */
 
 'use strict';
 
-export type DispatchConfig = any;
+import type {ReactInstance} from 'ReactInstanceType';
+
+export type DispatchConfig = {
+  dependencies: Array<string>,
+  phasedRegistrationNames?: {
```

Comment:
(We're both learning - that's the fun. :))

An example is the [EnterLeaveEventPlugin](https://github.com/facebook/react/blob/1229a238c491fbaf4c1596f82115b71524d0e56f/src/renderers/dom/client/eventPlugins/EnterLeaveEventPlugin.js#L19-L32) where the dispatch config has a registrationName and not a phasedRegistrationNames property.